src/EnergyPlus/Boilers.cc

```diff
@@ -299,6 +299,15 @@ void GetBoilerInput(EnergyPlusData &state)
         }
 
         thisBoiler.ParasiticElecLoad = state.dataIPShortCut->rNumericArgs(8);
+        thisBoiler.ParasiticFuelCapacity = state.dataIPShortCut->rNumericArgs(10);
+        if (thisBoiler.FuelType == Constant::eFuel::Electricity) {
```

Comment:
It looks like this is emitting a warning message every time the fuel type is electricity.  Regardless of the value of parasitic fuel capacity.  Even if the parasitic fuel capacity is entered as zero.  Am I reading it wrong?

Reply:
Oops, I think you're right. Let me double-check and fix it.